src/openforms/config/views.py

```diff
@@ -24,7 +23,7 @@ def _subset_match(requested: Optional[str], checking: str) -> bool:
     return requested == checking
 
 
-class ConfigurationView(UserIsStaffMixin, PermissionRequiredMixin, TemplateView):
+class ConfigurationView(PermissionRequiredMixin, UserIsStaffMixin, TemplateView):
```

Comment:
why was the order of these mixins changed?

Reply:
Just a minor change to avoid IDE warnings:

![image](https://user-images.githubusercontent.com/58147939/181040563-c8627c88-0cc0-450c-aec6-1795be031488.png)

Apparently it violates DJG301. I'm okay placing the builtin mixins first or last if there is some consistency.

Reply:
So apparently this is a PyCharm check (https://pycharm-security.readthedocs.io/en/latest/checks/DJG301.html) that isn't smart enough to see that the mixins to the left of it are also `AccessMixin`. Your IDE is giving wrong information in this case.

Don't blindly follow IDE warnings without understanding what you are changing, and don't fix what isn't broken. Let's revert this change.